portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,33 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Andrew Hojel's Porfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="writeSnippets()">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <center>
+          <img id="profile" src="images/profile_picture.jpg"/>
```

Comment:
please add an alt attribute to the image for screen reader users
https://webaim.org/techniques/alttext/

if the image doesn't provide meaning to the page, you can use alt="" to tell screen readers to skip over it.